### DISCUSSION TITLE
How to add style models and loras? where to download it? where to refer it?

### DISCUSSION THREAD

**Discussion by DisantHoridnt** [mrhan1993/Fooocus-API#348] [Q&A] [answered]
@mrhan1993 How to add style models and loras? where to download it? where to refer it?

**Comment by yak1ex** [comment, marked as answer]
To add style LORAs, you can follow a similar approach as the original Fooocus.
As stated in https://github.com/mrhan1993/Fooocus-API?tab=readme-ov-file#predownload-and-install , 
the default model files location is `repositories\Fooocus\models`, where you'll find a `loras` folder.
To add LORAs, simply download the required files and place them in this directory.
No additional configuration is required; however, you will need to specify LORAs in your API request.
You can also override these paths by specifying in `config.txt`, which is also described in the `README`.
